src/components/TextField/TextField.tsx

```diff
@@ -430,8 +430,10 @@ export function TextField({
     'aria-activedescendant': ariaActiveDescendant,
     'aria-autocomplete': ariaAutocomplete,
     'aria-controls': ariaControls,
-    'aria-multiline': normalizeAriaMultiline(multiline),
     'aria-expanded': ariaExpanded,
+    ...(normalizeAriaMultiline(multiline) && {
```

Comment:
How do you feel about changing the return type of the first case instead? https://github.com/Shopify/polaris-react/blob/a5cb04685a8404201cf15957bfdf2dedbd7f6504/src/components/TextField/TextField.tsx#L529